core/services/relay/evm/mercury/transmitter.go

```diff
@@ -269,6 +289,9 @@ func (mt *mercuryTransmitter) Transmit(ctx context.Context, reportCtx ocrtypes.R
 
 	mt.lggr.Debugw("Transmit enqueue", "req", req, "report", report, "reportCtx", reportCtx, "signatures", signatures)
 
+	if err := mt.orm.InsertTransmitRequest(req, reportCtx); err != nil {
+		return err
+	}
```

Comment:
This can grow unbounded in its current implementation; we probably don't want that

Reply:
We delete the row once the request has been transmitted to Mercury so it can only grow unbounded if no requests are being transmitted at all which shouldn't be a lot. What are you thinking about to mitigate this?

Reply:
I am imagining a scenario where the node gets disconnected from the mercury server for a long time. We don't want to write 100s of MB of data to transmissions table in this case.

We could implement pruning on this table so that it doesn't exceed (by much) `MaxTransmitQueueSize`. Alternatively we could delete a record when it is ejected from the queue due to overfilling (see: https://github.com/smartcontractkit/chainlink/blob/develop/core/services/relay/evm/mercury/queue.go#L86)

Reply:
Got it. I added a new goroutine that periodically prunes the table every 10s which should accumulate ~1000 or 10% of the MaxTransmitQueueSize. A batch delete places less load on the DB and we don't have to muddy up the interfaces of the queue and the ORM.